src/App.jsx

```diff
@@ -1,17 +1,53 @@
-import { useState } from 'react'
+import { useState, useEffect} from 'react'
 import './App.css'
+import Dashboard from './pages/Dashboard'
+import { Route, Routes } from 'react-router-dom'
+import PersonProfile from './pages/PersonProfile'
 
 export default function App() {
   const [hiredPeople, setHiredPeople] = useState([])
 
+  const [people, setPeople] = useState([])
+  useEffect(() => {
+    console.log("Fetching user...")
+    for (let i = 0; i < 50; i++)
```

Comment:
It would have been better to not call the API in a loop. You can pass the number of required users as a param, which will mean one network request rather than 50 :)